src/Microsoft.AspNetCore.Mvc.TagHelpers/CacheTagHelper.cs

```diff
@@ -132,17 +132,38 @@ public override int Order
         public TimeSpan? ExpiresSliding { get; set; }
 
         /// <summary>
-        /// Gets or sets the <see cref="CacheItemPriority"/> policy for the cache entry.
-        /// </summary>
-        [HtmlAttributeName(CachePriorityAttributeName)]
-        public CacheItemPriority? Priority { get; set; }
-
-        /// <summary>
         /// Gets or sets the value which determines if the tag helper is enabled or not.
         /// </summary>
         [HtmlAttributeName(EnabledAttributeName)]
         public bool Enabled { get; set; } = true;
 
+        /// <summary>
+        /// Gets or sets a key to discriminate cached entries.
+        /// </summary>
+        [HtmlAttributeName(KeyName)]
+        public string Key { get; set; }
+
+        /// <summary>
+        /// Additional options for the cache.
+        /// </summary>
+        [HtmlAttributeName(DictionaryAttributePrefix = OptionValuesPrefix)]
+        public IDictionary<string, string> OptionValues
+        {
+            get
+            {
+                if (_optionValues == null)
+                {
+                    _optionValues = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase);
```

Comment:
Because the ones in `CacheTagHelper` are. So why are they like this in `CacheTagHelper`, just a personal preference as they are taken from html attributes.